packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/DocumentController.java

```diff
@@ -98,8 +99,8 @@ public ResponseEntity<byte[]> downloadDocument(
 			.build()) {
 
 			final PresignedURL presignedURL = proxy.getDownloadUrl(id, filename).getBody();
-			final HttpPut put = new HttpPut(presignedURL.getUrl());
-			final HttpResponse response = httpclient.execute(put);
+			final HttpGet get = new HttpGet(presignedURL.getUrl());
```

Comment:
FANTASTIC CATCH :D

This was 100% a copy-paste error setting up the spring server and is everywhere in our code... I'll fix the other ones. thanks!